packages/date-picker/test/keyboard-input.test.js

```diff
@@ -15,8 +15,9 @@ describe('keyboard', () => {
     return datePicker._overlayContent.focusedDate;
   }
 
-  beforeEach(() => {
+  beforeEach(async () => {
     datePicker = fixtureSync('<vaadin-date-picker></vaadin-date-picker>');
+    await nextRender();
```

Comment:
Same as in `basic.test.js` - this change isn't necessary per se, but nice to have in terms of Lit.